File: API/Entities/PartnerService.cs
namespace API.Entities
{
    public class PartnerService
    {
        public int Id { get; set; }
        public string Status { get; set; }
        public int Year {get;set;}
        public string Note { get; set; }
        public Partner Partner {get;set;}
        public int? PartnerId {get;set;}
    }
}

File: client/src/app/_models/partnerservice.ts
export interface PartnerService {
    id:             number;
    status:         string;
    partnerId:      number;
    partnerName:    string;
    vendorId:       number;
    vendorName:     string;
    serviceId:      number;
    serviceName:    string;
    year:           number;
    note:           string;
    created?:        Date;
    lastEdited?:     Date;
    lastEditorId?:   number;
    statusUrl?:     string;
    gdprStatus?:    string;
    gdprStatusUrl?: string;
}
